deployment/helm/node-feature-discovery/templates/worker.yaml

```diff
@@ -54,6 +54,12 @@ spec:
         - "-key-file=/etc/kubernetes/node-feature-discovery/certs/tls.key"
         - "-cert-file=/etc/kubernetes/node-feature-discovery/certs/tls.crt"
 {{- end }}
+        - "-metrics={{ .Values.metrics.port }}"       
```

Comment:
Should we disable metrics here (i.e. set to 0) if `Values.metrics.enable=false`?

Same with master.